Mark URLPattern compare tests as tentative

jelee53 opened the pull request:
#### 12644dfca2c4e8c0ebe30aa523de9103f8d82172
<pre>
Mark URLPattern compare tests as tentative
<a href="https://bugs.webkit.org/show_bug.cgi?id=282162">https://bugs.webkit.org/show_bug.cgi?id=282162</a>
<a href="https://rdar.apple.com/138775980">rdar://138775980</a>

Reviewed by Sihui Liu.

URLPattern compare functionality is not in the URLPattern spec yet, see whatwg/urlpattern#61. Therefore, URLPattern compare tests should be marked tentative.

* LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.tentative.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.js.
(runTests):
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.serviceworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.serviceworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.serviceworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.serviceworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.sharedworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.sharedworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.sharedworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.sharedworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.worker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.worker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.worker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.serviceworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.serviceworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.serviceworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.serviceworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.sharedworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.sharedworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.sharedworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.sharedworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.worker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.worker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.worker.html.
* LayoutTests/platform/mac-wk1/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/285776@main">https://commits.webkit.org/285776@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7b0e61cac661dbba8594855f24d6fec89c361cf4

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/73789 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/53218 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/26600 "Built successfully") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/78111 "Failed to checkout and rebase branch from PR 35786") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/25027 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/62351 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/1003 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/5/builds/78111 "Failed to checkout and rebase branch from PR 35786") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/16411 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/76856 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/48184 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/63492 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/5/builds/78111 "Failed to checkout and rebase branch from PR 35786") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/44976 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/64/builds/20980 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/23360 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/66545 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/63/builds/21327 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/79678 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/1106 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/555 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/66389 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/1249 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/63503 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/65668 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/16222 "Built successfully and passed tests") | [  ~~🧪 vision-wk2~~](https://ews-build.webkit.org/#/builders/126/builds/9532 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/119/builds/7713 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/1070 "Built successfully") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/1099 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/1086 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/1105 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

szewai commented on LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/w3c-import.log:
```diff
@@ -15,7 +15,7 @@ None
 List of files:
 /LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-test-data.json
-/LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.js
+/LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.tentative.js
```
This file is auto generated during import and we should not modify it manually.

jelee53 committed:
Mark URLPattern compare tests as tentative
https://bugs.webkit.org/show_bug.cgi?id=282162
rdar://138775980

Reviewed by Sihui Liu.

URLPattern compare functionality is not in the URLPattern spec yet, see whatwg/urlpattern#61. Therefore, URLPattern compare tests should be marked tentative.

* LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.tentative.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/resources/urlpattern-compare-tests.js.
(runTests):
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.serviceworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.serviceworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.serviceworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.serviceworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.sharedworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.sharedworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.sharedworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.sharedworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.worker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.any.worker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.any.worker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.serviceworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.serviceworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.serviceworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.serviceworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.sharedworker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.sharedworker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.sharedworker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.sharedworker.html.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.worker-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.worker-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.tentative.https.any.worker.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/urlpattern/urlpattern-compare.https.any.worker.html.
* LayoutTests/platform/mac-wk1/TestExpectations:

Canonical link: https://commits.webkit.org/285776@main
